idd/Energy+.idd.in

```diff
@@ -31865,6 +31865,7 @@ ZoneHVAC:FourPipeFanCoil,
         \key CyclingFan
         \key VariableFanVariableFlow
         \key VariableFanConstantFlow
+        \key MultiStageFan
```

Comment:
@mjwitte No particular reason. I think, originally came from the user's request document.